lamindb/_curate.py

```diff
@@ -400,14 +472,15 @@ class AnnDataCurator(DataFrameCurator):
     See :doc:`docs:cellxgene-curate` for instructions on how to curate against a specific cellxgene schema version.
 
     Args:
-        data: The AnnData object or an AnnData-like path.
-        var_index: The registry field for mapping the ``.var`` index.
-        categoricals: A dictionary mapping ``.obs.columns`` to a registry field.
-        using_key: A reference LaminDB instance.
-        verbosity: The verbosity level.
-        organism: The organism name.
-        sources: A dictionary mapping ``.obs.columns`` to Source records.
-        exclude: A dictionary mapping column names to values to exclude.
+        data: `ad.AnnData | UPathStr` The AnnData object or an AnnData-like path.
```

Comment:
See above - not sure why we need type hints?

Reply:
I'm just seeing how it's done here: https://docs.lamin.ai/lamindb.artifact